demos/wasi/hello-world-c/README.md

```diff
@@ -55,3 +27,38 @@ CONFIG_MAP_VAL=cool stuff
 POD_NAME=hello-world-wasi-c
 []
 ```
+
+## Building from Source
+
+If you want to compile the demo and inspect it, you'll need to do the following.
+
+### Prerequisites
+
+Building WASI in C is easiest with the custom [SDK]() from wasmtime. Feel free to go down the rabbit hole of figuring
```

Comment:
So this was my mistake before (didn't include the link), but can you add this link to the `SDK` text? https://github.com/WebAssembly/wasi-sdk/releases

Reply:
fixed